obsplus/events/get_events.py

```diff
@@ -89,6 +89,8 @@ def _get_ids(df, kwargs) -> set:
         # Circular kwargs are used, first apply non-circular query then trim
         kwargs.update(_get_bounding_box(circular_kwargs))
         df = get_event_summary(df, **kwargs)
+        if len(df) == 0:  # If there are no events in the rectangular region.
+            return {}
```

Comment:
Apparently this doesn't effect anything but can you make the return value an empty set rather than a dict so the return type hint is correct?